src/services/BTCIdentifier/index.ts

```diff
@@ -4,13 +4,23 @@ import { IS_MAINNET } from '../../constants/common'
 
 const { BTC_TEST_IDENTIFIER } = config
 
-export const getBtcChainIdentify = async (txid: string) => {
+export const getBtcChainIdentify = async ({ txid, address }: { txid?: string; address?: string }) => {
   if (IS_MAINNET) {
     return 'mainnet'
   }
 
+  const params: Record<string, string> = {}
+
+  if (txid) {
+    params.txid = txid
+  }
+
+  if (address) {
+    params.address = address
+  }
+
   const identify = await axios
-    .get<{ chain: string }>(`${BTC_TEST_IDENTIFIER}/api/signet?${new URLSearchParams({ txid })}`)
+    .get<{ chain: string }>(`${BTC_TEST_IDENTIFIER}/api/signet?${new URLSearchParams(params)}`)
```

Comment:
A BTC address should be valid on both `testnet` and `signet`

Reply:
> A BTC address should be valid on both `testnet` and `signet`

Then which network should jump when click the link? @Sven-TBD

Reply:
> > A BTC address should be valid on both `testnet` and `signet`
> 
> Then which network should jump when click the link? @Sven-TBD

It depends, if the `btc address` appears in a `btc tx`(or the corresponding ckb tx), it should lead to the same network.
If the `btc address` is displayed standalonely, it can lead to the main testnet, `testnet 4` for now.

Reply:
> > > A BTC address should be valid on both `testnet` and `signet`
> > 
> > 
> > Then which network should jump when click the link? @Sven-TBD
> 
> It depends, if the `btc address` appears in a `btc tx`(or the corresponding ckb tx), it should lead to the same network. If the `btc address` is displayed standalonely, it can lead to the main testnet, `testnet 4` for now.

I think maybe it's better to return the btc env from the API

Reply:
> > > > A BTC address should be valid on both `testnet` and `signet`
> > > 
> > > 
> > > Then which network should jump when click the link? @Sven-TBD
> > 
> > 
> > It depends, if the `btc address` appears in a `btc tx`(or the corresponding ckb tx), it should lead to the same network. If the `btc address` is displayed standalonely, it can lead to the main testnet, `testnet 4` for now.
> 
> I think maybe it's better to return the btc env from the API

It's better but requires extra work on the backend @Sven-TBD